docs/en/observability/troubleshooting.asciidoc

```diff
@@ -17,13 +17,17 @@ Use the following tools in {kib} to troubleshoot issues across {observability} a
 [[troubleshooting-support]]
 === Elastic Support
 
+// tag::support[]
 We offer a support experience unlike any other.
 Our team of professionals 'speak human and code' and love making your day.
 https://www.elastic.co/subscriptions[Learn more about subscriptions].
+// end::support[]
 
 [float]
 [[troubleshooting-forum]]
 === Discussion forum
 
+// tag::discussion[]
 For other questions and feature requests,
-visit our https://discuss.elastic.co/c/observability[discussion forum].
\ No newline at end of file
+visit our https://discuss.elastic.co/c/observability[discussion forum].
```

Comment:
Looks like this is generically set for Observability.  Users might not realise to select _Synthetics_ from the drop down list, which would in effect create a forum post in https://discuss.elastic.co/c/observability/synthetics/75.